Add instrument names in tutorials  data section (#6101)
gammapy/gammapy: examples/tutorials/data/README.rst

Data exploration
================

> These tutorials show how to perform data exploration with Gammapy, providing an introduction to H.E.S.S., MAGIC, VERITAS, CTAO, HAWC and Fermi-LAT data and their instrument response functions (IRFs). You will learn to explore and filter event lists according to different criteria, as well as to get a quick look of the multidimensional IRFs files and perform instrument specific analysis.

Reviewer: ```suggestion
These tutorials show how to perform data exploration with Gammapy, providing an introduction to H.E.S.S., MAGIC, VERITAS, CTAO, HAWC and Fermi-LAT data and their instrument response functions (IRFs). You will learn to explore and filter event lists according to different criteria, view multidimensional IRF files and perform instrument-specific analysis.
```
